### DISCUSSION TITLE
URGENT - Very large quantity of Chia Available - We need help - Opportunity for Immediate Participation

### DISCUSSION THREAD

**Discussion by EarlyFarmer** [Chia-Network/chia-blockchain#17547] [Support]
To Whom it May Concern;

At the start of the CHIA project, my partner and engineer set us up to begin farming.  We were, at various times, rated between the largest and 5th largest validator and farmer.  Circumstances alter cases.  We are now rated between 20th and 30th largest node in the project.

We need help.  My engineer is not capable (due to circumstances) of figuring out how to take our personally mined and our validated CHIA off our servers and monetize it for the good of all.

We are looking for immediate help from someone who has the knowledge and skills to do so.  We are still connected to the network.  If anyone there can help we will figure out how to compensate as needed.

Please reach out to me via email at (REDACTED) or here. I am in the Pacific Time Zone.  If you are calling please text in advance so I know what Area Code you are calling from.

Server access is available.

Thank you.

Jeff

**Comment by BrandtH22** [comment]
Hey @EarlyFarmer , please note that I have redacted your email address from the initial message due to security reasons. 

There are a number of traditional exchanges listed on the community built chialinks site that provide offramp options (exchanging xch for other currencies): https://chialinks.com/exchanges/

Essentially you will likely want to work with an exchange to convert the xch into a form of currency that you can utilize for other purposes 👍

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
